### DISCUSSION TITLE
Full screen Alarm

### DISCUSSION THREAD

**Discussion by dharmikpatadia** [wmcmahan/react-native-calendar-events#373] [Q&A]
Can we able to set an full screen alarm like set by in built alarm clock?

**Comment by MoOx** [comment]
This feature won't fit in the goal of this lib.
Alarms that we handle here are for calendar event only, so the OS handle how to "display" them. User can even "ignore" some calendars, so you do not want to rely on this.

For standard OS alarm, you might need your own react-native bridge that could cover [Android AlarmManager](https://developer.android.com/reference/android/app/AlarmManager) (if I interpret this doc correctly), but [it doesn't seems possible for iOS](https://developer.apple.com/forums/thread/33824)

Alternatively, you can rely on [OS notification + some sound](http://andrewmarinov.com/building-an-alarm-app-on-ios/)
